eZ/Bundle/EzPublishRestBundle/behat.yml

```diff
@@ -0,0 +1,23 @@
+rest:
+    suites:
+        fullJson:
+            paths: [ vendor/ezsystems/ezpublish-kernel/eZ/Bundle/EzPublishRestBundle/Features ]
+            contexts:
+                - eZ\Bundle\EzPublishRestBundle\Features\Context\RestContext:
+                    url: http://localhost/api/ezp/v2/
```

Comment:
this can not hard code localhost anymore now that it is moved out.

So for this change we will need to do what I asked for long time ago, refactor to use relative url.